<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompassCalibrateControl</class>
 <widget class="QWidget" name="CompassCalibrateControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1220</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_txtMsg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>1211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_level">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 210, 165);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_back">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>231</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 210, 165);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_noseDown">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>140</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 210, 165);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_traildown">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 210, 165);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_left">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>370</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 210, 165);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_right">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>370</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 210, 165);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_calibration">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>620</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始校准</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_levelMsg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_backMsg">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>350</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_nosedownMsg">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>350</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_rightMsg">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>570</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_traildownMsg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_leftMsg">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>570</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Cancelcalibrate">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>620</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消校准</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_offset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>140</y>
     <width>311</width>
     <height>421</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="ck1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>610</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>磁1</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="ck2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>640</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>磁2</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
